domains/mkeko.json

```
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "Mkeko",
+           "email": "",
+           "discord": "1212043905059790881",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.IoL8azXADQL49lmOxbNtQkXU3iBoFgnPFo7dw-lTRdgyBLjlKIg7JZFO8D8DqgHOxQrNU-grdG0BCpSfBGHUmH2tQUP4Dwk_9_SuAPe7JCFUXhf3GcgUgb5odDaxbKwPg_ipB6KY3OGamFZM112IRjtlMMnFO91LZzXUBv1Sl8ZR8el8JnYtP4gqNKiIV0d2z-zcINAvUtWw03lu9VZbtItKWB3x6yRa_i2mxnrY4vRu6iYQfVBDhWHh6bS_ES_bZ_3h7DMi6EkGNJxCC_e3nmzq98lpAj0yxIKE-uvQSMpxnjm2c2PEv2ukp2fLbaUtAS9lQX90FMVQEurixmoI0w.CUvG4-bpkJamYuaRfMyAiw.K6bA2yqmMw1gBhus7qUKPWkacTae3ZzZ_eOCm9kw7zOrhd2P0H4i2FsWqjb6dpHOtXMxKjx34C8dxwBm_LLCx3mnTDShOETDlh83Kyazqls.M_T81AK6Z3kWZzocGp528w"
+        },
+    
+        "record": {
+            "CNAME": "https://mkeko.github.io/mkeko/"
```

```suggestion
            "CNAME": "https://mkeko.github.io"
```
CNAMEs cannot contain slashes `/`.